db-service/test/cqn4sql/flattening.test.js

```diff
@@ -377,6 +383,27 @@ describe('Flattening', () => {
       )
     })
   })
+
+  describe('in list', () => {
+    it('unfolds structure', () => {
+      let query = cqn4sql(
+        cds.ql`SELECT from bookshop.Books as Books {
```

Comment:
this is ok, would probably also work for assoc with multiple FKs or struc with multiple elems,
maybe add respecive test